runtime-config.md

```diff
@@ -1,6 +1,12 @@
 ## Mount Configuration
 
-Additional filesystems can be declared as "mounts", specified in the *mounts* array. The parameters are similar to the ones in Linux mount system call. [http://linux.die.net/man/2/mount](http://linux.die.net/man/2/mount)
+Additional filesystems can be declared as "mounts", specified in the *mounts* object.
+Keys in this object are names of mount points from portable config.
```

Comment:
Link with “`… of [mount points from the portable config](config.md#mount-points)`”?